hystrix-contrib/hystrix-metrics-event-stream/src/main/java/com/netflix/hystrix/contrib/metrics/eventstream/HystrixMetricsStreamServlet.java

```diff
@@ -135,7 +134,17 @@ private void handleRequest(HttpServletRequest request, HttpServletResponse respo
                 response.setHeader("Cache-Control", "no-cache, no-store, max-age=0, must-revalidate");
                 response.setHeader("Pragma", "no-cache");
 
-                MetricJsonListener jsonListener = new MetricJsonListener();
+                int queueSize = defaultMetricListenerQueueSize.get();
+                try {
+                    String q = request.getParameter("queueSize");
```

Comment:
If a system talks to multiple, dynamically scaleable endpoints, it could be cumbersome to change the queue size only via a property. Providing a request parameter would allow the hystrix-dashboard or other visualization service to be reconfigured in a much easier way.